config/initializers/assets.rb

```diff
@@ -1,10 +1,7 @@
 Rails.application.config.assets.precompile += %w(
-  miq_browser_detect.js
```

Comment:
yay! Cool that #6059 removed this from login.
I thought of it but didn't know if `application.js` was included on the login page. (sometimes people keep it separate on the login / home page)